x-pack/plugins/security_solution/public/detections/mitre/get_mitre_threat_component.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
This replaces the unnecessary code I had in `x-pack/plugins/security_solution/public/detections/pages/alert_details/tabs/summary/get_mitre_threat_component.ts` previously